src/vocutouts/uws/middleware.py

```diff
@@ -0,0 +1,33 @@
+"""Middleware for UWS services."""
+
+from typing import Awaitable, Callable
+from urllib.parse import urlencode
+
+from fastapi import Request, Response
+from starlette.middleware.base import BaseHTTPMiddleware
+
+
+class CaseInsensitiveQueryMiddleware(BaseHTTPMiddleware):
```

Comment:
Not really a problem, but I guess we're doing this so we don't have to put `uws_params_dependency` in every UWS handler?

Is it that `uws_params_dependency` is _only_ usable in POST handlers because's it's doing extra parsing? Sorry, just groking this UWS still.